apps/element-storybook/src/theme-provider.stories.tsx

```diff
@@ -2,11 +2,12 @@ import type { StoryObj } from '@storybook/react';
 import { ThemeProvider } from '@availity/theme-provider';
 
 export const Default: StoryObj<typeof ThemeProvider> = {
-  render: () => (
+  render: () => <>
     <ThemeProvider>
       <div>This component is inside the ThemeProvider</div>
     </ThemeProvider>
-  ),
+    <div>This component is outside the ThemeProvider</div>
```

Comment:
my changes will be putting every story in a ThemeProvider so this will still be styled.